mitosheet/css/toolbar.css

```diff
@@ -135,8 +134,21 @@
   font-family: Roboto;
 }
 
-.mito-toolbar-button-container-enabled:hover {
-  background-color: var(--mito-highlight-light);
+.mito-toolbar-button-container-enabled:hover .mito-toolbar-button-icon-container {
+  background-color: var(--mito-light-gray);
+}
+
+.mito-toolbar-tabbar-tabname span {
```

Comment:
I think these are still missing the :hover selectors for making it clear they are clickable, when closed.